### PR TITLE
added svelte.md svelte guide

### PR DESCRIPTION
**Pull request by RizkyRajitha**
NOTE:
- Please fill out the template below 

## Did you read the instructions in the [README](https://github.com/the-ethan-hunt/first-timers-guide/blob/master/README.md)?

> Y/N  yes

## Have you read the instructions in [CONTRIBUTING](https://github.com/the-ethan-hunt/first-timers-guide/blob/master/CONTRIBUTING.md)?

> Y/N yes

## Have you written this article by yourself? Have you taken inspiration from anywhere and then written it in your words?

>  i wrote it myself with aid of my article in medium about svelte . i also referred svelte official site to write this guide

### CODE REVIEW

File: guides/web_development/svelte.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# SVELTE Cybernetically enhanced web apps
+
+![svelte logo](https://miro.medium.com/max/1200/1*ldjKFBthtSWX-FidEupsFg.png "svelte logo")
+
+## WHAT EXACTLY SVELTE IS
+
+svelte got popular around 2019 with it's latest svetle 3 release.
```
Comment (the-ethan-hunt) [inline guides/web_development/svelte.md]: Please check for typos and implement proper grammar

File: guides/web_development/svelte.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+npm install
+npm run dev
+
+```
+
+![cli view](https://miro.medium.com/max/1375/1*ndEtKIKDLcy_O2-z2j2WVQ.png "cli view ")
+
+there you have it you successfully made hello world with svelte . 
+
+need somemore . visit my medium article on how to make a simple weather app with svelte
```
Comment (the-ethan-hunt) [inline guides/web_development/svelte.md]: No self-promotions!

**Comment by RizkyRajitha** [comment]
i made few changes , let me know if there are any issues.
thank you

**Comment by the-ethan-hunt** [comment]
Merging this PR. Great job @RizkyRajitha !

**Comment by RizkyRajitha** [comment]
thank you @the-ethan-hunt

**Pull request merged**